lib/report/lcovonly.js

```diff
@@ -75,9 +77,13 @@ Report.mix(LcovOnlyReport, {
             var branchArray = branches[key],
                 meta = branchMap[key],
                 line = meta.line,
-                i = 0;
+                i = 0,
+                covered, skipped;
+
             branchArray.forEach(function (b) {
-                writer.println('BRDA:' + [line, key, i, b].join(','));
+                covered = b > 0;
+                skipped = meta.locations && meta.locations[i] && meta.locations[i].skip;
+                writer.println('BRDA:' + [line, key, i, (covered || !skipped) ? b : 1].join(','));
```

Comment:
quite hare to read, maybe create a var on previous line for (covered || !skipped) ? b : 1